configuration_service/configuration_service.go

```diff
@@ -51,15 +52,17 @@ func (service ConfigurationService) StopProcessingRequests(ctx context.Context,
 	if err = service.authenticate("_StopProcessingRequests", request.Auth); err != nil {
 		return nil, err
 	}
-	return nil, fmt.Errorf("work in progress")
+	service.broadcast.trigger <- 0
```

Comment:
I would use constant names here and below instead of `0` and `1`, this will make code more readable